articles/confidential-computing/faq.md

```diff
@@ -27,32 +27,35 @@ If your Azure issue is not addressed in this article, visit the Azure forums on
 
 1. **Will all OS images work with Azure confidential computing?**
 
-   No. The virtual machines can only be deployed on Generation 2 virtual machines. We offer Generation 2 support for Ubuntu Server 18.04, Ubuntu Server 16.04, and Windows Server 2016 Datacenter. Read more about Gen 2 VMs on [Linux](../virtual-machines/linux/generation-2.md) and [Windows](../virtual-machines/windows/generation-2.md)
+   No. The virtual machines can only be deployed on Generation 2 virtual machines with Ubuntu Server 18.04, Ubuntu Server 16.04, and Windows Server 2016 Datacenter. Read more about Gen 2 VMs on [Linux](../virtual-machines/linux/generation-2.md) and [Windows](../virtual-machines/windows/generation-2.md)
 
 1. **DCsv2 virtual machines are grayed out in the portal and I can't select one**
 
-   Based on the information bubble next to the VM, there are different actions to take:
+   Make sure to toggle the filters as listed below and see if they become available:
     -	**UnsupportedGeneration**: Change the generation of the virtual machine image to “Gen2”.
     -	**NotAvailableForSubscription** : The region isn't yet available for your subscription. Select an available region.
-    -	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
+    
+1. **I get an ARM deployment failure error "Operation could not be completed as it results in exceeding approved standard DCSV2 Family Cores Quota**
+
+-	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
```

Comment:
"insufficient quota" is another "my vm is grayed out in the portal" issue.

This FAQ should stay - the ARM deployment error makes sense as well and the answer should still be to create a support request and increase quota, but this error **also** needs to stay in the answer to the question above as they describe two different ways of getting the same error

Reply:
yes it should go above as well.